### PR TITLE
Update PPP for milestone v1.4 (WH)

### PR DESCRIPTION
**Pull request by WeeeHung**
Added new contributions for v1.4 into own PPP

### CODE REVIEW

File: docs/team/weeehung.md [hunk 1]
```diff
@@ -51,9 +52,18 @@ Given below are my contributions to the project.
...
+    * Replaced all Screenshots of HealthSync GUI (v1.4) [\#228](https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/228)
+    * Added necessary breaklines to make sections more readable [\#228](https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/228)
+    * Added HealthSync Logo to User Guide [\#228](https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/228)
+    * Added `spacebar` key shortcut Tip to User Guide [\#228](https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/228)
+    * Added `Patient Card` in Glossary to orientate users on the fields present in the card [\#228](https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/228)
+
   * Developer Guide:
     * Added Product Scope which includes target user profile and value proposition.
     * Added user stories and sort them in terms of priority
+    * Updated UML Diagrams for UI Component to include LoggedPesonListPanel [\#252](https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/252)
```
Comment (WZWren) [inline docs/team/weeehung.md:64]:
```suggestion
    * Updated UML Diagrams for UI Component to include LoggedPersonListPanel [\#252](https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/252)
```

File: docs/team/weeehung.md [hunk 2]
```diff
@@ -63,6 +73,8 @@ Given below are my contributions to the project.
 
 * **Tools**:
   * Utitlised JavaFx Scene Builder for building the frontend of the project ([\#107](https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/107))
```
Comment (WZWren) [inline docs/team/weeehung.md:75]:
```suggestion
  * Utilised JavaFx Scene Builder for building the frontend of the project ([\#107](https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/107))
```

File: docs/team/weeehung.md [hunk 3]
```diff
@@ -63,6 +73,8 @@ Given below are my contributions to the project.
...
+  * Utitlised Markbind CLI to preview PUML diagrams and web changes locally [\#252](https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/252)
```
Comment (WZWren) [inline docs/team/weeehung.md:76]:
```suggestion
  * Utilised Markbind CLI to preview PUML diagrams and web changes locally [\#252](https://github.com/AY2324S1-CS2103T-T14-3/tp/pull/252)
```

File: docs/team/weeehung.md (hunk 1 above)
Comment (WeeeHung) [reply docs/team/weeehung.md:64]: Thanks for spotting the typo

File: docs/team/weeehung.md (hunk 2 above)
Comment (WeeeHung) [reply docs/team/weeehung.md:75]: Thanks for spotting the typo!

**Pull request merged**